packages/slate-editor/src/components/EditorBlock/EditorBlock.tsx

```diff
@@ -136,17 +135,6 @@ export const EditorBlock = forwardRef<HTMLDivElement, Props>(function (
         [isOnlyBlockSelected],
     );
 
-    useEffect(() => {
-        const onEsc = (e: KeyboardEvent) => {
```

Comment:
I've raised this to Al and he agreed to get rid of it, but I think having the ability to close the menu is not really useful since the block still stays selected afterwards.

So I think it's better to keep the menu open as long as the block is selected.

Reply:
JFYI It's no longer possible to close the menu of the site gallery aside after this change.

https://github.com/user-attachments/assets/ad887935-4cd7-4bd6-8bf8-50a5036af5c0

Reply:
> That's because the gallery block is still selected. When deselected, the menu would close automatically.
However because there's no other node to select, you can't really deselect it.

Yep.

It's a little bit annoying :)